staging/src/k8s.io/kubectl/pkg/cmd/delete/delete_test.go

```diff
@@ -972,3 +972,50 @@ func TestResourceErrors(t *testing.T) {
 		})
 	}
 }
+
+func TestDeleteMessageOutput(t *testing.T) {
+	cmdtesting.InitTestErrorHandler(t)
+	_, _, rc := cmdtesting.TestData()
+
+	tf := cmdtesting.NewTestFactory().WithNamespace("test-specific")
+	defer tf.Cleanup()
+
+	codec := scheme.Codecs.LegacyCodec(scheme.Scheme.PrioritizedVersionsAllGroups()...)
+
+	tf.UnstructuredClient = &fake.RESTClient{
+		NegotiatedSerializer: resource.UnstructuredPlusDefaultContentConfig().NegotiatedSerializer,
+		Client: fake.CreateHTTPClient(func(req *http.Request) (*http.Response, error) {
+			switch p, m := req.URL.Path, req.Method; {
+			case p == "/namespaces/test-specific/replicationcontrollers/redis-master" && m == "DELETE":
+				return &http.Response{StatusCode: http.StatusOK, Header: cmdtesting.DefaultHeader(), Body: cmdtesting.ObjBody(codec, &rc.Items[0])}, nil
+			default:
+				t.Fatalf("unexpected request: %#v\n%#v", req.URL, req)
+				return nil, nil
+			}
+		}),
+	}
+
+	streams, _, buf, _ := genericiooptions.NewTestIOStreams()
+	cmd := NewCmdDelete(tf, streams)
+	err := cmd.Flags().Set("filename", "../../../testdata/redis-master-controller.yaml")
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	cmd.Run(cmd, []string{})
+
+	if !strings.Contains(buf.String(), "replicationcontroller") {
+		t.Errorf("message doesn't contain type (replicationcontroller): %s", buf.String())
+	}
+
+	if !strings.Contains(buf.String(), "deleted") {
+		t.Errorf("message doesn't contain action (deleted): %s", buf.String())
+	}
+
+	if !strings.Contains(buf.String(), "redis-master") {
+		t.Errorf("message doesn't contain name (redis-master): %s", buf.String())
+	}
+
+	if !strings.Contains(buf.String(), "test-specific") {
+		t.Errorf("message doesn't contain namespace (test-specific): %s", buf.String())
+	}
```

Comment:
I think adding one condition that compares complete message is easier to maintain instead of these 4 `Contains`. What do you think?